Assets/Vimeo/Scripts/Config/VimeoVideo.cs

```diff
@@ -100,6 +100,12 @@ public int CompareTo(VimeoVideo other)
             return 0;
         }
 
+        public TextAsset GetDescriptionAsTextAsset()
+        {
+            string tempMeta = description.Replace("\\n", "");
```

Comment:
This method has no tests and the function is very brittle. It's assuming that the description is JSON, but what happens when it is a mix of text and json? Should use regex to parse it out of the body.

And the function should then be renamed to something like `GetMetadataAsTextAsset` which then calls another method called `GetMetadata` which does the regex/json parsing, both of which should have tests.